### PR TITLE
add hints (#153)

### PR DESCRIPTION
**Pull request by gizemkeser**
Associated Issue: #153 
### Summary of Changes

- hint implementation
- add/update tests

### CODE REVIEW

**Comment by gizemkeser** [comment]
I will resolve the conflict

**Comment by ollelauribostrom** [comment]
@gizemkeser Nice job, I'll have a look at this tomorrow 🙂

File: src/js/components/rebus.js [hunk 1]
```diff
@@ -19,20 +20,31 @@ export function Rebus(props, ...children) {
...
+          !showHint ? animation : 'none'
+        }">
             <div class="rebus__header">
               <span>${current + 1}/${rebuses.length}</span>
             </div>
             <span class="rebus__symbols">${rebus.symbols.join(' ')}</span>
             <div class="rebus__words">
               <children>
             </div>
+            ${
```
[the remark this answers is not in the document]
Comment (gizemkeser) [reply src/js/components/rebus.js]: @ollelauribostrom thanks for your feedback, it's done.

**Comment by ollelauribostrom** [comment]
Nice job, thank you 🙂

**Pull request merged**